### DISCUSSION TITLE
[IMPORTANT] NoiseModelling Day - 9th June 2021

### DISCUSSION THREAD

**Discussion by pierromond** [Universite-Gustave-Eiffel/NoiseModelling#332] [General]
:mega: If you see this message, you are invited to participate to the “**NoiseModelling day 2021**” on **June 9th, 2021 from 10am to 4pm (French, UTC+2)** in the form of a webinar.

In 2020, we organized the first NoiseModelling days in the form of a general presentation of the tool and tutorials.

This year we will focus on the user:

- The morning will be dedicated to the presentation of the 2021 novelties and to a series of user presentations in "flash" format (5 min + 3 min questions) or "classic" format (15 min + 5 min questions).
- In the afternoon, we will continue user presentations and discuss future development needs.
- The final goal is to federate a community around NoiseModelling and to better meet the needs of users.

Participation in the seminar is **free of charge** but **registration is required** at the following link:  [https://forms.gle/svkB9NixcN6Kpodc6](https://forms.gle/svkB9NixcN6Kpodc6)

**Presenters must register by May 1, 2021 and other participants by May 23, 2021.**

Best regards,

The NoiseModelling team

**Comment by pierromond** [comment]
@ioannides-dB @tomasanda @annezhangxue @merghd @Floriangr  etc.

**Reply by Floriangr** [reply]
Awesome, thanks a lot for the invite! Looking forward to it.

**Reply by ioannides-dB** [reply]
Thank you for the invitation. Let me see what I can do.

**Comment by gpetit** [comment]
And if needed, to spread the "official" word, there is an "official" web page here : https://noise-planet.org/noisemodelling_days.html
